vispy/visuals/isosurface.py

```diff
@@ -42,6 +43,9 @@ def __init__(self, data=None, level=None, vertex_colors=None,
         self._recompute = True
         self._update_meshvisual = True
 
+        shading = kwargs.pop('shading', None)
```

Comment:
Can't we add this kwarg to the `__init__` declaration?

Reply:
Given the comment below, this change wouldn't be needed anymore. The shading filter would be part of the `MeshVisual` the `shading` param forwarded to it in the `kwargs`. (As before the changes, if I understand well.)